cvat-ui/src/reducers/notifications-reducer.ts

```diff
@@ -390,26 +397,69 @@ export default function (state = defaultState, action: AnyAction): Notifications
                 },
             };
         }
-        case AuthActionTypes.ACCEPT_INVITATION_SUCCESS: {
+        case InvitationsActionTypes.GET_INVITATIONS_SUCCESS: {
+            if (action.payload.showNotification) {
+                return {
+                    ...state,
+                    messages: {
+                        ...state.messages,
+                        invitations: {
+                            ...state.messages.invitations,
+                            newInvitations: 'You have new organization invitations. [Click here](/invitations) to view them.',
```

Comment:
I would suggest something simpler: 

Somebody invited you to an organization. [Click here](/invitations) to get details